enhancements/etcd/etcd-encryption-for-separate-oauth-apis.md

```diff
@@ -103,39 +102,79 @@ To validate if `CAO` is capable of managing its own `encryption-config` we are g
   
 Note: the above tests will be created based on the common test library that drives the same set of tests for `OAS-O` and `KAS-O`.
 
-To validate upgrade / downgrade path for `4.n` and `4.(n-1)` we are going to create the following E2E tests:
- - scenario 1: take `4.(n-1)` cluster, encrypt it, upgrade to `4.n`, downgrade and upgrade again
- - scenario 2: take `4.(n-1)` cluster, encrypt it, upgrade to `4.n`, force key rotation, downgrade to `4.(n-1)`, force key rotation and upgrade again
+To validate upgrade / downgrade path for `4.6` and `4.5` we are going to manually check the following E2E cases:
+
+Scenario 1: 
```

Comment:
I tested it manually yesterday. Didn't find any issues with our components except missing clean up logic (ns and the new conditions). I will open a new PR today.

I did run into some issues with other components, `openshift-samples` and `monitoring` went degraded during the downgrade.